exporter/util.go

```diff
@@ -218,17 +218,24 @@ func (ic *importContext) IsUserOrServicePrincipalDirectory(path, prefix string,
 }
 
 func (ic *importContext) emitRepoByPath(path string) {
-	ic.Emit(&resource{
-		Resource:  "databricks_repo",
-		Attribute: "path",
-		Value:     strings.Join(strings.SplitN(path, "/", 5)[:4], "/"),
-	})
+	parts := strings.SplitN(path, "/", 5)
```

Comment:
This code was cryptic before and it is still cryptic. Can you add a comment with an example of why this string manipulation is necessary?

Reply:
It's to create a base repos path... This part is covered by tests, but it will change when we start to support Git folders.

Reply:
I see, it's not immediately obvious to me why we are splitting by 5, can you also mention an example of before and after split in the comments as well. Do we always stop splitting after 4th `/`? This seems a bit fragile if there is change in how path is represented on backend but I guess there isn't a better way to do this?